tests/third_party/cupy/math_tests/test_rounding.py

```diff
@@ -10,7 +10,7 @@
 @testing.gpu
 class TestRounding(unittest.TestCase):
     @testing.for_all_dtypes(no_complex=True)
-    @testing.numpy_cupy_allclose(atol=1e-5)
+    @testing.numpy_cupy_allclose(type_check=False, atol=1e-5)
```

Comment:
`type_check=False` was added here since `dpctl` (following array API) returns the same type of input for `ceil`, `floor`, and `trunc` while `numpy` always returns floating-point number. For instance, if input is integer `numpy` returns `float` but `dpctl` returns `integer`, and the data type comparison test fails.

Reply:
Ok, now I see. I previously assumed that `type_check=False` is caused only by another dtype returned in case of device without fp64 aspect.
Then please keep the current implementation and ignore the above suggestion.